packages/proxy/src/providers/bedrock.ts

```diff
@@ -341,14 +350,48 @@ export async function fetchConverse({
     } else {
       toolConfig = translateTools(parsed.data);
     }
-    delete rest.functions;
+    delete oaiParams.functions;
+  }
+
+  let isStructuredOutput = false;
+  const parsed = z
+    .object({
+      type: z.literal("json_schema"),
+      json_schema: responseFormatJsonSchemaSchema,
+    })
+    .safeParse(oaiParams.response_format);
+  if (parsed.success) {
+    isStructuredOutput = true;
+    if (toolConfig) {
+      throw new ProxyBadRequestError(
+        "Structured output is not supported with tools",
+      );
+    }
+    toolConfig = {
```

Comment:
don't you need to delete the `response_format` param? Maybe converse doesn't barf?

Reply:
yes, will do, although looks like it doesn't go crazy

Reply:
Oh, just realized we don't use oaiParams anymore. It's translated, not modified